vimeo-networking/src/main/java/com/vimeo/networking/model/Spatial.java

```diff
@@ -72,4 +72,34 @@ public Format getFormat() {
                 return Format.UNKNOWN;
         }
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Spatial spatial = (Spatial) o;
+
+        return mProjection.equals(spatial.mProjection) && mStereoFormat.equals(spatial.mStereoFormat);
```

Comment:
Stag will guarantee that they are `NotNull`. However, if we construct the object outside of the stag environment, then we cannot guarantee it. This is an interesting problem, I will add this to the sync agenda next week